slack-base/src/main/java/com/hubspot/slack/client/models/interaction/blockkit/BlockKitLoadOptionsRequestIF.java

```diff
@@ -0,0 +1,23 @@
+package com.hubspot.slack.client.models.interaction.blockkit;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.interaction.InteractiveLoadOptionsRequestType;
+import com.hubspot.slack.client.models.response.views.ViewResponseBase;
+import com.hubspot.slack.client.models.teams.SlackTeam;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@HubSpotStyle
+@JsonNaming(value = PropertyNamingStrategy.SnakeCaseStrategy.class)
+public interface BlockKitLoadOptionsRequestIF {
```

Comment:
Can we use this instead https://github.com/HubSpot/slack-client/blob/master/slack-base/src/main/java/com/hubspot/slack/client/models/interaction/BlocksLoadOptionsRequestIF.java?

Reply:
Negative, it contains almost all but the `view` field is absent. It contains callbackId, which we're using in handle() method.

Reply:
Can we extend the existing model with an optional `view` field?